templates/results.html

```diff
@@ -15,17 +15,112 @@ <h5 style="color: white;">{{ e.name }} to {{  e.target["name"] }} </h5>
   		{{ e.body }}
   		</textarea>
   		<!-- <button class="btn btn-dark" role="button" onclick="_{{ loop.index }}()">Send to {{ e['email'] }}</button> -->
-  		<a class="btn btn-dark mb-1" href="mailto:{{e.target['email']}}?subject={{e.mailto_subject}}&body={{e.mailto_body}}" role="button">Open in default email app</a>
+  		<a class="btn btn-dark mb-1" id="default-mail-button" href="mailto:{{e.target['email']}}?subject={{e.mailto_subject}}&body={{e.mailto_body}}" role="button">Open in default email app</a>
 
-  		<a class="btn btn-danger btn-email mb-1" onclick="copy_compose('gmail', {{loop.index}}, '{{e.target['email']}}', '{{e.mailto_subject}}')" role="button">Open in Gmail</a>
-  		<a class="btn btn-primary btn-email mb-1" onclick="copy_compose('outlook', {{loop.index}}, '{{e.target['email']}}', '{{e.mailto_subject}}')" role="button">Open in Outlook</a>
-  		<a class="btn btn-info btn-email mb-1" onclick="copy_compose('yahoo', {{loop.index}}, '{{e.target['email']}}', '{{e.mailto_subject}}')"role="button">Open in Yahoo Mail</a>
+  		<a class="btn btn-danger btn-email mb-1" data-toggle="modal" data-target="#newsletterModal" onclick="copy_compose('gmail', {{loop.index}}, '{{e.target['email']}}', '{{e.mailto_subject}}')" role="button">Open in Gmail</a>
```

Comment:
Why are the data-toggle and data-target fields necessary to be added on the buttons to send a template with a specific email provider?

Reply:
the data-toggle and data-target trigger the modal pop up to display, I could make so that there is a listener for these buttons instead but that seems to add unnecessary js code.

Reply:
Thanks for clarifying on that, agree it's preferable to a listener, however I was questioning what the effect will be - is it so that when someone clicks "Open in Gmail" they get a popup asking them to subscribe to the mailing list? Because that doesn't seem ideal and wouldn't capture people on mobile (for whom these buttons don't show, but make up a large chunk of the users).

Reply:
@rafeeJ since you've tried it out perhaps you could address this concern? otherwise very happy to approve it for staging